src/main/kotlin/solve/utils/ResourcesUtils.kt

```diff
@@ -59,3 +62,16 @@ fun loadBufferedImage(filePath: String): BufferedImage? {
 
     return image
 }
+
+fun createAlert(content: String, owner: Window) {
```

Comment:
May be another file in that folder for the function. It is not about resources usage